logic_mod/Blocks/BlockUtils.cs

```diff
@@ -44,6 +44,17 @@ public static bool TryGetLong(object arg, out long value)
             return false;
         }
 
+        public static bool TryGetBool(object arg, out bool value)
+        {
+            value = false;
+            if (arg is long ilev)
+            {
+                value = ilev != 0;
+                return true;
+            }
+            return false;
+        }
+
```

Comment:
Please make TryGetBool logic similar to "CheckCond" function from Script/Block.cs. JS should be able to cast bool, long, float, undefined and any object to bool, so that func([1,2,3]) would be identical to func(true) if it is expecting bool as argument.